src/main.js

```diff
@@ -868,17 +868,16 @@ textAngular.service('textAngularManager', ['taToolExecuteAction', 'taTools', 'ta
 		},
 		// this is used by taBind to send a key command in response to a special key event
 		sendKeyCommand: function(scope, event){
-			angular.forEach(editors, function(_editor){
```

Comment:
Yeah. In the case of tab this is correct. I'm wondering if sendKeyCommand is used for some other purpose that should be triggered in all instances. A for like that doesn't go without a reason. Any thoughts? 

Anyway, I'll search for where it is used and see how it is used. Give me a few minutes.